TwoWayRecyclerView/genericepg/src/main/java/genericepg/duna/project/view/EPGView.java

```diff
@@ -198,8 +198,8 @@ public boolean dispatchTouchEvent(MotionEvent ev) {
 
     private <T extends View> void dispatchEventForView(MotionEvent ev, T genericView) {
         //genericView.getLocationInWindow(location);
-        int height = timelineRecyclerView.getHeight();
-        int width = channelsRecyclerView.getWidth();
+        int height = genericView.getHeight();
```

Comment:
by doing so, is the programs clicked correctly when click near the margins? Have to take into account the left width of channels logo and height of top timeline, then dispatch the recalculated event position to the subviews. This commit is not fixing the sporadic issue with vertical alignment

Reply:
The fix will imply the LayoutManager

Reply:
Oh, I see. Thank you!

Reply:
Did you find a solution?

Reply:
I realized that the problem is not on the surface and have not yet dived into details.